seqr/views/utils/pedigree_info_utils.py

```diff
@@ -437,7 +438,8 @@ def _parse_datstat_export_format(rows):
 
 
 def _get_datstat_family_notes(row):
-    row = {k: unicode(v, errors='ignore') for k, v in row.items()}
+    row = {k: v.encode('ascii', errors='ignore').decode() if isinstance(v, str) else str(v, 'ascii', errors='ignore')
```

Comment:
if it is an instance of `str`, wouldn't `str(v, 'ascii', errors='ignore')` still work? so why add the complicated conditional?

Reply:
If it is an instance of Python 3 `str` then `str(v, 'ascii', errors='ignore')` will cause an error as below:
```
>>> v='abc'
>>> type(v)
<class 'str'>
>>> str(v, 'ascii', errors='ignore')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: decoding str is not supported
>>> 
```
The `str(v, 'ascii', errors='ignore')` works without errors for Python 2 but won't strip out unicodes for `unicode/newstr` instances:
```
>>> str_s='abc'
>>> str_s_with_unicode='\xe2abc'
>>> unicode_s=u'\xe2abc'
>>> newstr_s=str(str_s)
>>> newstr_s_with_unicode=str(unicode_s)
>>> type(str_s)
<type 'str'>
>>> type(str_s_with_unicode)
<type 'str'>
>>> type(unicode_s)
<type 'unicode'>
>>> type(newstr_s)
<class 'future.types.newstr.newstr'>
>>> type(newstr_s_with_unicode)
<class 'future.types.newstr.newstr'>
>>> str(str_s, 'ascii', errors='ignore')
'abc'
>>> str(str_s_with_unicode, 'ascii', errors='ignore')
'abc'
>>> str(unicode_s, 'ascii', errors='ignore')
'\xe2abc'
>>> str(newstr_s, 'ascii', errors='ignore')
'abc'
>>> str(newstr_s_with_unicode, 'ascii', errors='ignore')
'\xe2abc'
>>> 

```